src/miner_sup.erl

```diff
@@ -42,7 +42,7 @@ start_link() ->
 init(_Args) ->
     SupFlags = #{
         strategy => rest_for_one,
-        intensity => 2,
+        intensity => 10,
```

Comment:
Why was this bumped to 10?

Reply:
the onion server was crashing because it was handling msgs before the chain was fully up, it was a startup timing issue.  The restart strategy here was ineffective, in that 2 did not give the much wriggle room for the chain to come back up before the next restart of the onion server and thus we hit a breach of the sup intensity and the entire tree was restarted.

Effectively with a value of 2/5 the rest_for_one strategy is no better than a one_for_all.   So this change is to give it that little bit of additional grace before we take down the entire miner tree.

Reply:
If we've fixed the crash, does this still matter?

Reply:
It doesnt matter in relation to this issue being addressed but same issue could occur in other scenarios.  I am fine with removing the intensity change if there is concern about it, but the fact that the miner tree can easily be taken down is something to be aware off

Reply:
Miner crashing is fine, runit will restart it. I'd prefer we retract this change for now, the rest of the PR seems fine.